config/config_test_reload_error.go

```diff
@@ -0,0 +1,89 @@
+// +build all
```

Comment:
is it worth adding `!race` here so it's clear we don't want to run this with the race detector?